testing/run_tests.py

```diff
@@ -133,7 +133,13 @@ def RunCCTests(build_dir, filter):
   # TODO(44614): Re-enable after https://github.com/flutter/flutter/issues/44614 has been addressed.
   # RunEngineExecutable(build_dir, 'fml_unittests', filter, [ fml_unittests_filter ] + shuffle_flags)
 
-  RunEngineExecutable(build_dir, 'runtime_unittests', filter, shuffle_flags)
+  # TODO(tbd): Remove this explicit testing of release builds after lightweight
+  # isolates are supported in JIT mode.
+  release_runtime_out_dir = EnsureReleaseRuntimeTestsAreBuilt()
+  RunEngineExecutable(release_runtime_out_dir, 'runtime_unittests', filter, shuffle_flags)
```

Comment:
according to https://cs.opensource.google/flutter/recipes/+/master:recipes/engine.py;l=684?q=file:recipes%2Fengine.py, we might not have to do this

Reply:
We need run_tests.py able to run locally too.  I would have expected the build to be a noop, but looks like the goma check might be happening before it releases it doesn't need to build something.

Reply:
in that case, wouldn't you do `run_tests.py --type=engine --variant=host_release_unopt`?

Reply:
I tried `host_release_unopt` I removed that after reading through the recipe, I think it's using `host_release` https://cs.opensource.google/flutter/recipes/+/master:recipes/engine.py;l=683?q=file:recipes%2Fengine.py

Reply:
I'm going to put a try block around the build so that it can continue to execution on the server, but hopefully still build locally.